src/components/VmDetails/cards/DisksCard/DiskListItem.js

```diff
@@ -95,29 +93,23 @@ const DiskListItem = ({
           storageDomainList={storageDomainList}
           onSave={onEdit}
           trigger={
-            <OverlayTrigger
-              overlay={<Tooltip id={`${idPrefix}-action-edit-tooltip`}>{msg.diskEditTooltip()}</Tooltip>}
-              placement='left'
-            >
+            <OverlayTooltip id={`${idPrefix}-action-edit-tooltip`} tooltip={msg.diskEditTooltip()} placement='left'>
```

Comment:
As I see `placement='left'` is the most popular, so I think it will be better to make this as default value, for `placement`

Reply:
`placement` can be removed as well as down below, because `left` is default value.